pkg/cloudprovider/vsphere/cloud.go

```diff
@@ -89,11 +91,17 @@ func (vs *VSphere) Initialize(clientBuilder cloudprovider.ControllerClientBuilde
 	} else {
 		klog.Errorf("Kubernetes Client Init Failed: %v", err)
 	}
+	if vs.loadbalancer != nil {
+		vs.loadbalancer.Initialize(client, stop)
+	}
 }
 
 // LoadBalancer returns a balancer interface. Also returns true if the
 // interface is supported, false otherwise.
 func (vs *VSphere) LoadBalancer() (cloudprovider.LoadBalancer, bool) {
+	if vs.loadbalancer != nil {
```

Comment:
Can we also gate this feature with an env var like `ENABLE_ALPHA_NSXT_LB`. I want to make it really clear to users that this is an alpha feature.